samples/msgext-link-unfurling/csharp/TeamsApp/launchSettings.json

```diff
@@ -0,0 +1,13 @@
+﻿{
+  "profiles": {
+    // Debug project within Teams
+    "Microsoft Teams (browser)": {
+      "commandName": "Project",
+      "launchUrl": "https://teams.microsoft.com/l/app/%TEAMSAPPID%?installAppPackage=true&webjoin=true&appTenantId=%TENANTID%&login_hint=%USERNAME%"
```

Comment:
The `launchUrl` seems wrong. Please use `${{TEAMS_APP_ID}}` replacing `%TEAMSAPPID%`, also, check other variables and `launchUrl`.

Reply:
Fixed